### PR TITLE
feat: register docker-compose cli for compose extension

### PR DESCRIPTION
**Pull request by cdrage**
feat: register docker-compose cli for compose extension

### What does this PR do?

Update to https://github.com/containers/podman-desktop/pull/4388

Adds the docker-compose CLI to the list of CLI's available in the CLI
page

image

### What issues does this PR fix or reference?

Related to #3781
Depends on #4197

### How to test this PR?

Check the CLI section within settings and you'll see Compose

Signed-off-by: Charlie Drage <charlie@charliedrage.com>

### CODE REVIEW

**Comment by cdrage** [comment]
~https://github.com/containers/podman-desktop/pull/4387 MUST be merged before this PR~ Merged now.

No tests added to this PR since this is a simple addition of the CLI extension to the activate section.

**Comment by cdrage** [comment]
@dgolovin ready for review 👍

**Review by dgolovin — APPROVED** [review]
Added screenshot to description.
LGTM.

**Review by feloy — APPROVED** [review]

File: extensions/compose/src/extension.ts
Comment (benoitf) [inline extensions/compose/src/extension.ts:39]: hello, I'm not sure about the part around `docker compose`

For sure you can run `docker-compose` CLI but `docker compose` if you don't have `docker` CLI or compose being at a right place it'll not work

Comment (benoitf) [reply extensions/compose/src/extension.ts:39]: here I think you should use @mairin text from https://github.com/containers/podman-desktop/pull/4534 ?

Comment (benoitf) [inline extensions/compose/src/extension.ts:39]:
```suggestion
const composeDescription = `The Compose extension provides optional command line support for [compose files](https://compose-spec.io/) with Podman.\n\nMore information: [Podman Desktop Documentation](https://podman-desktop.io/docs/tags/compose)`;
```

**Review by benoitf — CHANGES_REQUESTED** [review]
please update the compose description

**Comment by cdrage** [comment]
Blocked on your review, can you please re-approve again @benoitf ?

**Review by benoitf — APPROVED** [review]

**Pull request merged**
